docs/release_notes/v0.10.0.md

```diff
@@ -421,14 +421,14 @@ app.get('/dapr/subscribe', (req, res) => {
         {
             topic: "newOrder",
             route: "orders",
-            pubsubName: "messagebus"
+            pubsubname: "messagebus"
```

Comment:
This is case sensitive, have you tried it? Also, we are using camelCase, not?

Reply:
this works either way, this is for consistency across all the repos in the Dapr org.

Reply:
Tehcnicallty, this was in release notes for the version we are not patching so maybe leave this file as is

Reply:
Are we moving away from camelCase? `newOrder` here is still in camelCase.

Reply:
The [API](https://github.com/dapr/docs/blob/master/reference/api/pubsub_api.md#http-response-body) has to be the source of truth there, no?

Reply:
The documentation is mixed. The code should be the source of truth and it uses camelCase. I agree to bring consistency - which seems to be to stick to camelCase.

https://github.com/dapr/dapr/blob/ce89980eda9cc86efbc11a745e2959b1925236ca/pkg/runtime/runtime.go#L73

Reply:
paging @yaron2, I know he had some opinions on this topic

Reply:
> The documentation is mixed

Only a very few places had this camelCase BTW. Also, worth pointing out that this is for REST API only. The protos have number of places where they differ from the HTTP API docs.

Reply:
When I tested this, both `pubsubname` and `pubsubName` were valid.

Reply:
@artursouza this is the correct place: https://github.com/dapr/dapr/blob/4e3198ca8d84184b77083674a2018ac2ad76b484/pkg/runtime/pubsub/subscription.go#L4

Reply:
The reference you gave is about internal metadata that dapr sets for itself, and unrelated to the `pubsubname` the user sets.

Reply:
In this example, we also need to fix "newOrder" to "neworder"